CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 
 ### Fixes
 
+- `[pretty-format]` Fixed a bug where "anonymous custom elements" were not being printed as expected. ([#15138](https://github.com/jestjs/jest/pull/15138))
```

Comment:
Could you put it in alphabetical order? (On mobile, or I'd do it myself)

Reply:
🤦 my bad. Will do.

Reply:
@SimenB Done